capstone/src/main/webapp/youtubeScript.js

```diff
@@ -154,9 +166,11 @@ function showAvailableAttributes() {
   });
 }
 
-/** Returns the sum of all elements in an array */
-arrSum = function(arr) {
-  return arr.reduce(function(a, b) {
-    return a + b
-  }, 0);
+/** Checks if a character is a letter */
+function isLetter(character) {
```

Comment:
I think we should avoid using hard-coded character values, as these can always change in the future and make the code more difficult to understand. In my opinion, regular expressions are cleaner to use, as with something like this [example](https://www.w3resource.com/javascript/form/all-letters-field.php).